bench/workgen/runner/example_dynamic_tables.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+# This workload simulates the continuous creation of tables.
+
+import threading as pythread
+import random
+import string
+import time
+
+from runner import *
+from wiredtiger import *
+from workgen import *
+
+
+class ThreadWithReturnValue(pythread.Thread):
+
+    def __init__(self, group=None, target=None, name=None, args=(), kwargs={}):
+        pythread.Thread.__init__(self, group, target, name, args, kwargs)
+        self._return = None
+
+    def run(self):
+        if self._target is not None:
+            self._return = self._target(*self._args, **self._kwargs)
```

Comment:
You've written this class to be reusable, which is great!  But here, if the _target is none, it silently returns without an error.  Maybe either throw an exception if target is None?  Or not make the check and let it crash?

Reply:
Thanks, that's a good point. I removed the check on the target, see here 945261a65b24e903208eec6990edfc8a054c591c.
@ruby-oujo, let us know if you wanted to do something differently.

Reply:
Thanks for addressing this @etienneptl, sounds good to me.